src/Fantomas.Core/paket.references

```diff
@@ -1,3 +1,4 @@
 FSharp.Core
+System.Collections.Immutable
```

Comment:
I'm a bit conflicted about this one. The ImmutableQueue makes sense and is nice, but at the same time, it does introduce an additional dependency.